vhdbuilder/packer/install-dependencies.sh

```diff
@@ -164,23 +164,28 @@ for CRICTL_VERSION in ${CRICTL_VERSIONS}; do
 done
 
 installAndConfigureArtifactStreaming() {
-  # download acr-mirror proxy
+  # arguments: package name, package extension
+  PACKAGE_NAME=$1
+  PACKAGE_EXTENSION=$2
   MIRROR_PROXY_VERSION='0.2.3'
-  UBUNTU_VERSION_CLEANED="${UBUNTU_RELEASE//.}"
-  MIRROR_DOWNLOAD_PATH="./acr-mirror-${UBUNTU_VERSION_CLEANED}.deb"
-  MIRROR_PROXY_URL="https://acrstreamingpackage.blob.core.windows.net/bin/${MIRROR_PROXY_VERSION}/acr-mirror-${UBUNTU_VERSION_CLEANED}.deb"
-  
-  retrycmd_curl_file 10 5 60 $MIRROR_DOWNLOAD_PATH $MIRROR_PROXY_URL || exit ${ERR_ARTIFACT_STREAMING_DOWNLOADL}
-  
-  apt_get_install 30 1 600 $MIRROR_DOWNLOAD_PATH || exit $ERR_ARTIFACT_STREAMING_DOWNLOAD
-
-  rm "./acr-mirror-${UBUNTU_VERSION_CLEANED}.deb"
+  MIRROR_DOWNLOAD_PATH="./$1.$2"
+  MIRROR_PROXY_URL="https://acrstreamingpackage.blob.core.windows.net/bin/${MIRROR_PROXY_VERSION}/${PACKAGE_NAME}.${PACKAGE_EXTENSION}"
+  retrycmd_curl_file 10 5 60 $MIRROR_DOWNLOAD_PATH $MIRROR_PROXY_URL || exit ${ERR_ARTIFACT_STREAMING_DOWNLOAD}
+  if [ "$2" == "deb" ]; then
+    apt_get_install 30 1 600 $MIRROR_DOWNLOAD_PATH || exit $ERR_ARTIFACT_STREAMING_DOWNLOAD
+  elif [ "$2" == "rpm" ]; then
+    dnf_install 30 1 600 $MIRROR_DOWNLOAD_PATH || exit $ERR_ARTIFACT_STREAMING_DOWNLOAD
+  fi
+  rm $MIRROR_DOWNLOAD_PATH
 }
 
 UBUNTU_MAJOR_VERSION=$(echo $UBUNTU_RELEASE | cut -d. -f1)
 if [ $OS == $UBUNTU_OS_NAME ] && [ $(isARM64)  != 1 ] && [ $UBUNTU_MAJOR_VERSION -ge 20 ]; then
-  # install and configure artifact streaming
-  installAndConfigureArtifactStreaming || exit $ERR_ARTIFACT_STREAMING_DOWNLOAD
+  installAndConfigureArtifactStreaming acr-mirror-${UBUNTU_RELEASE//.} deb
+fi
+
+if [ $OS == $MARINER_OS_NAME ]  && [ $OS_VERSION == "2.0" ]; then
```

Comment:
Is there an arm64 version of the package or should this be skipped for isARM64 like on Ubuntu?

Reply:
Skipped ARM 64 build since we don't have it for Ubuntu and Azure Linux